WcaOnRails/config/locales/en.yml

```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
+      'regulations1':
+        title: "Is my puzzle competition legal?"
+        #context: Answer to "Is my puzzle competition legal?"
+        content:
+          '1': "You can find the regulations regarding puzzles in <a href='https://www.worldcubeassociation.org/regulations/#3'>Article 3</a>. All puzzles have to be approved by the WCA Delegate of the competition. If you are unsure, please ask him. To be on the safe side, you should bring a replacement puzzle in case you are not allowed to use it."
```

Comment:
Just a thought about the "him", since the regulations make such an effort to remain gender neutral (https://github.com/thewca/wca-regulations/issues/40 and https://github.com/thewca/wca-regulations/issues/288), maybe we should strive for the same here?